src/System.IO.Compression.Brotli/System.IO.Compression.Brotli.csproj

```diff
@@ -12,24 +15,54 @@
     <OutputTypeEx>library</OutputTypeEx>
     <StartupObject />
   </PropertyGroup>
-  <!-- Platform specific properties -->
-  <PropertyGroup Condition="'$(Platform)' == 'amd64'">
+  <PropertyGroup Condition=" '$(Platform)' == 'x64'">
```

Comment:
A "Chose/When" construct might be clearer:
https://github.com/dotnet/corefx/blob/master/dir.props#L133-L150

You could also add the PropertyGroup and ItemGroup under one condition.

Currently, there are duplicate checks:
```csproj
<PropertyGroup Condition=" '$(OS)' == 'Windows_NT'"> 
<ItemGroup Condition=" '$(OS)' != 'Windows_NT'"> 
```

Thanks @mellinoe for the recommendation.

@Vedin, thoughts?